app/views/items/menu.html.erb

```diff
@@ -1,2 +1,49 @@
-<%= render 'shared/notice' %>
-<h1>Welcome</h1>
\ No newline at end of file
+<div class="container-fluid mt-4 px-5">
+  <%= render 'shared/notice' %>
+  <div class="d-flex align-items-center mb-4">
+    <h1 class="fw-bold">Menu</h1>
+    <span class="mx-5">
+      <%= form_tag menu_path, method: :get do |form| %>
+        <div class="form-check form-check-inline">
+          <%= radio_button_tag :non_veg, 'nil', params[:non_veg]&.eql?('nil') || params[:non_veg].nil?, class: "form-check-input", onclick: "this.form.submit()"%>
+          <%= label_tag :non_veg_nil, "All", class:"form-check-label" %>
+        </div>
+        <div class="form-check form-check-inline">
+          <%= radio_button_tag :non_veg, false, params[:non_veg] == 'false', class: "form-check-input", onclick: "this.form.submit()"%>
+          <%= label_tag :non_veg_false, "Veg", class:"form-check-label" %>
+        </div>
+        <div class="form-check form-check-inline">
+          <%= radio_button_tag :non_veg, true, params[:non_veg] == 'true', class: "form-check-input", onclick: "this.form.submit()" %>
+          <%= label_tag  :non_veg_true, "Non-Veg", class:"form-check-label" %>
+        </div>
+      <% end %>
+    </span>
+    <h2 class="ms-auto"><%= @branch.name %></h2>
+  </div>
+  <div class="row row-cols-2 row-cols-md-3 row-cols-lg-4">
+    <% @meals.each do |meal| %>
+      <div class="col mb-4">
+        <div class="card">
+          <% if meal.image.attached? %>
+            <%= image_tag meal.image, class: 'card-img-top', style: 'height: 250px; object-fit: cover;', alt: meal.name %>
+          <% else %>
+            <%= image_tag 'coming-soon.jpg', alt: 'CHAMPS', style: 'height: 250px; object-fit: cover;', alt: 'Image Coming Soon' %>
+          <% end %>
+          <div class="card-body">
+            <h5 class="card-title"><%= meal.name %>
+              <svg xmlns="http://www.w3.org/2000/svg" width="20" height="20" fill= "<%= meal.non_veg? ? 'green' : 'red' %>" class="bi bi-stop-circle" viewBox="0 0 16 16">
```

Comment:
Also, last request
Please check correct symbols for veg non veg. It is the square outside and filled circle inside.
Please fix and align it in the right most side (above add to cart)

Reply:
Done